### DISCUSSION TITLE
"Setup boilerplate" not visible in my repo's Action tab

### DISCUSSION THREAD

**Discussion by Lionel-Wilson** [DevSkillsHQ/discusssions#46] [Help] [answered]
I'm following the instructions for getting set up. And I can't seem to see any "Setup Boilerplate" nor is there any left panel. Are these instructions outdated?

**Comment by fenske** [comment, marked as answer]
Hi @Lionel-Wilson. I'm Anton from Alva.

Recorded this Loom for you: https://www.loom.com/share/7936d60b73394509889f4d2bcb771ca3

Could you check it out?

**Reply by Lionel-Wilson** [reply]
That's so weird. I don't have that left panel. but it's okay now. I managed to do it manually. thank you :)

**Reply by fenske** [reply]
Hm. Interesting. 🤔 If you have a minute and you could share a screenshot of how it looks to you, I'd kindly appreciate it. 🙏

Otherwise, I wish you the best of luck with the rest of your interview process!

**Reply by Lionel-Wilson** [reply]
![image](https://github.com/DevSkillsHQ/discusssions/assets/80872669/a55e378e-ee15-4026-bd24-29b7822a11d2)

**Reply by fenske** [reply]
Judging by the screenshot, it seems you're in the wrong repository, i.e. https://github.com//Lionel-Wilson/Invar-Group-Lookup-service.

And here's your assignment's repo that came in Alva's invite: https://github.com/ffc1e12/lookup-service-level-1_7ab7a9d-ac9gcf (this is where you should create a PR once you're done)
